src/config.rs

```diff
@@ -249,6 +249,14 @@ pub struct Config {
 
     /// Path where config was loaded from
     config_path: Option<PathBuf>,
+
+    /// Enable visual bell
+    #[serde(default="true_bool")]
+    visual_bell: bool,
+
+    /// Visual bell duration in milliseconds
+    #[serde(default="default_visual_bell_duration")]
+    visual_bell_duration: u16
```

Comment:
The config for this would be simplified by just having a `visual_bell_duration: Option<u16>`. The `visual_bell` flag would no longer be needed.